src/botpress.js

```diff
@@ -142,10 +144,19 @@ class botpress {
       postgres: botfile.postgres
     })
 
-    const security = createSecurity({
+    const cloud = await createCloud({ projectLocation, botfile, logger })
+
+    if (!!botfile.login.useCloud && (await cloud.isPaired())) {
```

Comment:
@slvnperron , I think using `Boolean(botfile.login.useCloud)` is prefered. At least I had this rule in eslint by default earlier. :)

Reply:
Can you explain why this is preferred? In fact there's an implicit boolean coercion so the explicit cast is unnecessary

Reply:
I've seen styleguides claiming that double-negation is an obscurity. But that's more a matter of preference so we can leave it as is.

Reply:
Matter of style, obviously. It's obscure but also well-recognized.